README.md

```diff
@@ -26,6 +26,9 @@
       --host=<host>    If starting server, specify ip address to bind to.
                        Default is 127.0.0.1
       --port=<port>    Specify server port number. Default is 9288
+      --remote-trigger Defer printing until page evaluates the following:
+                       window.cefPdf({request: "", onSuccess: function () {}, onFailure: function () {}});
```

Comment:
This would allows the client to decide when to give up rather than having to implement this inside `cef-pdf`.